src/db/schemas/pollSchema.js

```diff
@@ -1,7 +1,6 @@
 const mongoose = require('mongoose')
 
 module.exports = new mongoose.Schema({
-    userId: String,
     serverId: String,
-    polls: String
+    polls: { pollId: String, pollTitle: String, pollCreatorId: String }
```

Comment:
This is a serious breaking change that completely wrecks past polls(I suppose in this option as feature is relatively new and not fully fledged out we can just wipe out past records instead of adjusting to them), also it would be good to include creation timestamp here too.
![image](https://user-images.githubusercontent.com/11031051/135767815-504170a8-e54a-4fcf-8d3d-a1205999462c.png)

Reply:
Implemented the timestamp.
![image](https://user-images.githubusercontent.com/81253203/135898903-1596ef25-3742-458e-bdcf-18e2f01b29b4.png)